zhaquirks/tuya/tuya_smoke.py

```diff
@@ -20,7 +22,16 @@
 
 (
     TuyaQuirkBuilder("_TZE204_ntcy3xu1", "TS0601")
+    .applies_to("_TZE200_ntcy3xu1", "TS0601")
     .tuya_smoke(dp_id=1)
+    .tuya_binary_sensor(
+        dp_id=4,
+        attribute_name="tamper",
+        device_class=BinarySensorDeviceClass.TAMPER,
+        entity_type=EntityType.DIAGNOSTIC,
+        translation_key="tamper",
```

Comment:
Similar to the [Hue tamper sensor](https://github.com/zigpy/zha-device-handlers/blob/b20b82bdc7f3702e67c037f3e966aed97b061d05/zhaquirks/philips/soc001.py#L65-L72), we can remove the `translation_key` here to use the device class translation in Home Assisstant.

```suggestion
```